part_1/title_v/sekcja_31_PL.md

```diff
@@ -0,0 +1,32 @@
+## <a name="title_5"></a><p align="center">T Y T U Ł &emsp; V<br /> LICENCJA &emsp; LUB &emsp; POZWOLENIE &emsp; NA &emsp; BROŃ &emsp; PALNĄ</p>
+
+### <a name="section_31"></a><p align="center">Artykuł 31</p>
+
+Licencje lub pozwolenia na broń palną i ich Grupy
+
+Licencja lub pozwolenie na broń to oficjalne zezwolenie dla osoby prawnej lub fizycznej uprawniające do nabycia i posiadania broni palnej i amunicji w ramach zezwolenia określonego dla poszczególnych grup licencji lub pozwolenia na broń palną lub do wykonywania działań w zakresie niewybuchów zgodnie z pozwoleniem określonym dla licencji lub pozwolenia dla grupy K.
+
+Licencje lub pozwolenia na broń są podzielone,
+zgodnie z celem używania broni palnej i amunicji oraz zakresu zezwolenia, na następujące grupy:
+
+a) A - opracowywanie i produkcję broni palnej i amunicji;
+
+b) B - naprawy, modyfikacje lub dezaktywację broni palnej i amunicji;
+
+c) C - nabywanie, sprzedaż oraz transport broni palnej i amunicji;
+
+d) D - wypożyczanie lub przechowywanie broni palnej i amunicji;
+
+e) E - niszczenie lub dezaktywacja broni palnej i amunicji;
+
+f) F - szkolenie lub nauka strzelania;
+
+g) G - ochrona mienia i osób;
+
+h) H - organizowanie lub bycie gospodarzem imprez sportowych, kulturowych i podobnych;
+
+i) I - prowadzenie działań w zakresie dziedzictwa kulturowego, lub w celach kolekcjonerskich;
+
+j) J - prowadzenie działań na podstawie specjalnego rozporządzenia<sup>[10)](#fn10)</sup>; and
```

Comment:
Choć nie mam jeszcze absolutnej pewności, to `special legal regulation` wydaje mi się być po prostu `rozporządzeniem`. Przypis odwołuje się do rozporządzenia o straży gminnej.

Reply:
Definicja z [jakiejś wiki](http://wiki.knihovna.cz/index.php/Pr%C3%A1vn%C3%AD_p%C5%99edpisy): "Zvláštní právní předpis - takový termín označuje odkaz na jiný právní předpis, než v ten, ve které je odkaz uveden. Pokud tedy součástí konkrétního zákona bude norma, která se odkazuje na jiný (například dříve vydaný) předpis, pak se označí jako zvláštní." --> z google translate: Specjalna regulacja prawna - termin ten dotyczy odesłania do prawa innego niż to, w którym dokonano odniesienia. Jeśli więc część prawa szczególnego jest normą, która odnosi się do innej (na przykład wydanej wcześniej) recepty, to jest oznaczona jako specjalna.